src/modules/fw_mode_manager/fw_mode_manager_params.c

```diff
@@ -580,6 +580,20 @@ PARAM_DEFINE_INT32(FW_LND_ABORT, 3);
  */
 PARAM_DEFINE_INT32(FW_LAUN_DETCN_ON, 0);
 
+/**
+ * Pre-launch throttle
+ *
+ * Throttle value set before launch detection and until the motor delay is up.
```

Comment:
You might want to note what cases this is for. 

Are other docs updates required? Some places they might be considered
- https://docs.px4.io/main/en/flight_modes_fw/takeoff
- https://docs.px4.io/main/en/flight_modes_fw/mission#mission-takeoff
- https://docs.px4.io/main/en/config_fw/position_tuning_guide_fixedwing